### ISSUE TITLE
Comments on product page

### ISSUE THREAD

**Issue by edvinnn** [SpyTec/D0018E-ShopStore#14]
- [x] Back-end

- [x] Front-end to work properly

- [ ] Testing

**Comment by edvinnn** [comment]
@SpyTec Ska man endast kunna kommentera om man har köpt produkten?

**Comment by SpyTec** [comment]
Nej, vi kör att bara inloggade kan göra det, oavsett om de köpt något eller inte
